src/stories/index.stories.js

```diff
@@ -412,6 +448,48 @@ storiesOf('Components', module)
             template: `<div style="width: 400px"><PaymentInfoLine :amount="${amount}" :fee="${fee}" origin="${origin}"/></div>`,
         };
     })
+    .add('SendTx', () => {
+        return {
+            components: {SendTx, SmallPage, IdenticonContact},
+            template: windowTemplate(`
+                <small-page style="height: 560px;">
+                    <SendTx address="NQ10 7" :disabled="false">
```

Comment:
false is the default for `disabled` already.

Also I think, SendTx should only accept valid addresses.

Might also want to make the address as knob.